src/api.ts

```diff
@@ -44,6 +44,7 @@ export default class Chromeless<T extends any> implements Promise<T> {
         closeTab: true,
         ...options.cdp,
       },
+      chromeFlags: ['--headless'],
```

Comment:
Why not `['--headless', ...(options.chromeFlags||[])]` ?

Reply:
I also like the object hash option